student.rb

```diff
@@ -1,11 +1,19 @@
 require './person'
+require './classroom'
 
 class Student < Person
+  attr_accessor :classroom
```

Comment:
- [ ] I suggest that you only allow the read-option for the student-classroom association. This is because no writing is done by the classroom class on the student class, it only checks/reads their status.